packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -18,6 +18,7 @@
 .usa-checkbox {
   max-width: 320px;
   background: transparent;
+  padding-bottom: 1px;
```

Comment:
Can you add this to the Storybook example instead? Adding it here means we're diverging from USWDS styles.

Reply:
This was just a test, and then it passed after removing this, so we should just keep an eye on [this test](https://www.chromatic.com/test?appId=65a6e2ed2314f7b8f98609d8&id=65ea3bc591ce6dd1397ab13f) in the future. Updating the storybook makes a lot more sense.